PocketKit/Sources/PocketKit/MyList/SavesContainerViewController.swift

```diff
@@ -385,7 +385,7 @@ extension SavesContainerViewController {
 
     private func present(tagsFilterViewModel: TagsFilterViewModel?) {
         guard true, let tagsFilterViewModel = tagsFilterViewModel else { return }
-        let hostingController = UIHostingController(rootView: TagsFilterView(viewModel: tagsFilterViewModel))
+        let hostingController = UIHostingController(rootView: TagsFilterView(viewModel: tagsFilterViewModel).environment(\.managedObjectContext, Services.shared.source.viewContext))
```

Comment:
what about adding this the view itself?

Reply:
I tried, it didnt work. I think because adding it to the view is too late.